qa/tasks/cephfs/test_admin.py

```diff
@@ -1412,15 +1412,15 @@ def test_multifs_single_path_rootsquash(self):
         """
         Test root_squash with multi fs
         """
+        if not isinstance(self.mount_a, FuseMount):
```

Comment:
I think this can be removed altogether. Is there a reason for this @kotreshhr ?

Reply:
@vshankar

Reply:
@kotreshhr ping

@vshankar QA was successful for this PR.

Reply:
@kotreshhr ping

@vshankar QA was successful for this PR.

Reply:
@kotreshhr ping? IMO this test is valid for kclient.

Reply:
Yeah, rootsquash is valid for kclient too. I tested that in recent main branch. But I wasn't sure why this check was present in ```test_single_path_rootsquash``` , I added that for multifs too. 

As @vshankar pointed out, we can remove this check altogether in both single fs and multi fs tests.

Reply:
Got it. Removed and updated the commit message. Thanks!